pyo3-build-config/src/impl_.rs

```diff
@@ -458,19 +477,11 @@ print("mingw", get_platform().startswith("mingw"))
 
     #[allow(clippy::unnecessary_wraps)]
     pub fn fixup_import_libs(&mut self) -> Result<()> {
```

Comment:
Isn't the function name a misnomer now as it only generates the import libraries now?

Reply:
How about `generate_import_libs`? Or any other suggestions?

Reply:
The suggested name sounds fine to me. I wonder if the cfg flags can be simplified, e.g. pub on the outside of the function instead using blocks?